### DISCUSSION TITLE
Scores of symbols not updated

### DISCUSSION THREAD

**Discussion by M-Stenzel** [rspamd/rspamd#5102] [General]
Hi folks, I run rspamd 3.8.4 on Linux, non-dockerised.

I am pretty pretty satisfied with rspamd, but...

I tried to change the score of symbols (via the web interface, not by changing a config file), but the result of the rspamd scan do not reflect these changes. And yes, I did restart rspamd several times (maybe this wasn't even necessary).

So here we go:

![image](https://github.com/user-attachments/assets/05c4bd30-4921-47d1-a93f-70c3a31bfa4a)

I changed the weight of the URIBL_GREY to 3.00, as you can see below

![image](https://github.com/user-attachments/assets/5615f7aa-0cf5-4fa5-997d-910a47a1c609)

But this change obviously did not get applied. 
Even more interestingly I have this result:

rspamadm configdump -g (only relevant part, others deleted)

```
URIBL_GREY {
            score = 1.500000;
            description = "A domain in the message is listed in URIBL.com grey";
            disabled = false;
            one_shot = true;
            extra_groups [
                "surbl",
                "uribl",
            ]
        }


URIBL_GREY { 
            score = 1.500000;
            description = "A domain in the message is listed in URIBL.com grey";
            disabled = false;
            one_shot = true;
            extra_groups [
                "uribl",
                "rbl",
            ]
        }
```

There are no warnings (no errors) in the logs.

**Comment by feld** [comment]
I'm experiencing this on FreeBSD with rspamd 3.11.1

I increased the value of a symbol significantly and it still shows it in the web interface as being the custom value I provided but when I scan it does not use that value and the config dump also shows the original value. Very confusing.

**Comment by moisseev** [comment]
First of all, `configdump` is unaware of the dynamic configuration, so you can't check settings defined in the WebUI with it.

Actually, it seems Rspamd is using the score from the WebUI (the default in that Rspamd version was URIBL_GREY=1.5, which matches the `configdump` output). I'd rather ask why it doubles the score even though `one_shot = true;` is configured. I'd suggest checking the Rspamd log (maybe bump up the debug level for the module) – it often has clues about why things don't work as expected.

**Reply by cweiske** [reply]
Where are dynamic configuration values stored? I'd like to move those to a new server.

**Reply by cweiske** [reply]
Found them: `/var/lib/rspamd/rspamd_dynamic` - see https://github.com/rspamd/rspamd/discussions/4654 and https://docs.rspamd.com/faq/#what-should-i-back-up
